### PR TITLE
Add some features

### PR DESCRIPTION
**Pull request by sudh-0**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Sample file.
+Some changes.
 Changes from the original.
```
Comment (sudh-0) [inline readme.md:3]: Why these changes?

Comment (tsuki42) [reply readme.md:3]: Some appropriate reply

File: readme.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 Sample file.
+Some changes.
```
Comment (tsuki42) [inline readme.md:2]: This can be done instead of that

**Review by tsuki42 — CHANGES_REQUESTED** [review]
